bin/assemble_FITS

```diff
@@ -35,60 +35,63 @@ def cmdline():
     parser.add_argument("--outname", action="store", default=None,
                         help="The name of the output FITS file to be created")
-    parser.add_argument("--byext", action="store_true",default=False, 
+    parser.add_argument("--byext", action="store_true", default=False,
                         help="Sort by externsion name rather than list order?")
-    parser.add_argument("--dtype", action="store",default=numpy.int32,
+    parser.add_argument("--dtype", action="store", default=numpy.int32,
                         help="The 'dtype' for the image arrays (default=int32)")
-    parser.add_argument("--compress_type", action="store",default=None, choices=compress_choices,
+    parser.add_argument("--compress_type", action="store", default=None, choices=compress_choices,
                         help="Compress type for FITS file, default")
 
-    #args = parser.parse_args()
+    # args = parser.parse_args()
     # Set the defaults of argparse using the values in the yaml config file
     parser.set_defaults(**conf_defaults)
     args = parser.parse_args(args=remaining_argv)
     return args
 
-def elapsed_time(t1,verb=False):
-    t2    = time.time()
-    stime = "{:d}m {:2.2f}s".format( int( (t2-t1)/60.), (t2-t1) - 60*int((t2-t1)/60.))
+
+def elapsed_time(t1, verb=False):
+    t2 = time.time()
+    stime = "{:d}m {:2.2f}s".format(int((t2-t1)/60.), (t2-t1) - 60*int((t2-t1)/60.))
     if verb:
         print("Elapsed time: {}".format(stime))
     return stime
 
-def read_pixels(filename,shape,dtype=numpy.int32):
+
+def read_pixels(filename, shape, dtype=numpy.int32):
     """ Read in the pixels from a file"""
     fd = open(filename, 'rb')
     img_data = numpy.fromfile(file=fd, dtype=numpy.int32).reshape(shape)
     return img_data
 
+
 def assemble_FITS(**kwargs):
 
     """Simple function to assemble header file with pixels written in raw binary format"""
 
     # Extract the kwrags from the function
-    header_file = kwargs.get('header_file',None)
-    image_list = kwargs.get('image_list',None)
-    naxis1   = kwargs.get('naxis1', 0)
-    naxis2   = kwargs.get('naxis2', 0)
-    outname  = kwargs.get('outname',None)
-    byext =    kwargs.get('byext',False)
-    dtype =    kwargs.get('dtype',numpy.int32)
-    compress_type = kwargs.get('compress_type',None)
+    header_file = kwargs.get('header_file', None)
+    image_list = kwargs.get('image_list', None)
+    naxis1 = kwargs.get('naxis1', 0)
+    naxis2 = kwargs.get('naxis2', 0)
+    outname = kwargs.get('outname', None)
+    byext = kwargs.get('byext', False)
+    dtype = kwargs.get('dtype', numpy.int32)
+    compress_type = kwargs.get('compress_type', None)
 
     # Make sure that we have sane values for NAXIS1/NAXIS1
     if naxis1 == 0 or naxis2 == 0:
-        print("WARNING: Both NAXIS1/NAXIS2 muste be non-zero, current values are:{},{}".format(naxis1,naxis2))
-    
+        print("WARNING: Both NAXIS1/NAXIS2 muste be non-zero, \
```

Comment:
Backslash continuation within a quoted string is unusual and probably does not have the effect desired (particularly with regard to spacing).  Concatenation is more typical.

Reply:
Fixed